packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -138,6 +138,9 @@ abstract class FlutterCommand extends Command<Null> {
   /// tracking of the command.
   Future<String> get usagePath async => name;
 
+  /// Additional usage values to be sent with the usage ping.
+  Future<Map<String, String>> get usageValues async => const <String, String>{};
```

Comment:
the other variable was usagePath, so I kept with "usage". We could refactor both names?